finagle-core/src/main/scala/com/twitter/finagle/param/Params.scala

```diff
@@ -128,14 +128,17 @@ object HighResTimer {
    *
    * It is a shared resource and as such, `stop` is ignored.
    */
-  private[finagle] val Default: com.twitter.util.Timer =
-    new JavaTimer(true, Some("HighResTimer"))
+  val Default = new JavaTimer(true, Some("HighResTimer")) {
+    override def stop(): Unit = ()
+    def stopTimer(): Unit = super.stop()
```

Comment:
it won't compile
```
[info] Compiling 1 Scala source to C:\Users\GQ28BL\Projects\work\ing\arch\okapi\finagle\finagle\finagle-core\target\scala-2.12\classes ...
[error] C:\Users\GQ28BL\Projects\work\ing\arch\okapi\finagle\finagle\finagle-core\src\main\scala\com\twitter\finagle\param\Params.scala:141:13: value stopTimer is not a member of com.twitter.util.JavaTimer
[error]     Default.stopTimer()
[error]             ^
[error] one error found
[error] (finagle-core / Compile / compileIncremental) Compilation failed
[error] Total time: 22 s, completed 2019-10-10 10:13:17
```

Reply:
ok, well this is a bit convoluted, but we can make it work with

```
private[this] val underlying = new JavaTimer... {
  ...
  def stopTimer...
}

val Default: JavaTimer = underlying

def stop(): Unit = underlying.stopTimer
```

and let's add some kind of note that we're doing this because we don't want it to be accidentally stoppable.